generateInitialKey/generateInitialKey.js

```diff
@@ -38,8 +39,9 @@ function generateAddressCallback(keyObject, password) {
 async function attachToContract(initialKey, cb) {
 	let config = getConfig();
 	let web3 = await configureWeb3(config);
-	let contractABI = config.Ethereum.contracts.KeysManager.abi;
-	let contractAddress = config.Ethereum.contracts.KeysManager.addr;
+	let contractABI = KeysManagerAbi;
+	const network = process.env.NETWORK;
+	let contractAddress = config.Ethereum.contracts.KeysManager[network]['addr'];
```

Comment:
Same remark as in https://github.com/poanetwork/poa-scripts-validator/pull/9 applies here